docs/usage/configuration/config_documentation.md

```diff
@@ -904,6 +904,23 @@ Example configuration:
 redaction_retention_period: 28d
 ```
 ---
+---
+### `purge_retention_period`
```

Comment:
Since https://github.com/matrix-org/synapse/issues/13476 isn't solved yet, we should probably add an unsafe warning somewhere here.

Reply:
No, if the room doesn't have local users in it it's safe from what I understood of the problem.